test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityPCLTests.cs

```diff
@@ -222,11 +222,17 @@ public void CompatibilityPCL_OptionalUnk(string fw1, string fw2)
         [InlineData("portable-net45+win8+monoandroid+monotouch", "portable-net45+win8+wp8")]
         [InlineData("portable-net45+win8+monoandroid1+monotouch1", "portable-net45+win8+wp8")]
         [InlineData("monoandroid10", "portable-net45+win8")]
+        [InlineData("monoandroid10", "portable-net451+netcore451")]
         [InlineData("monotouch10", "portable-net45+win8")]
+        [InlineData("monotouch10", "portable-net451+netcore451")]
         [InlineData("xamarinios10", "portable-net45+win8")]
+        [InlineData("xamarinios10", "portable-net451+netcore451")]
         [InlineData("xamarintvos10", "portable-net45+win8")]
+        [InlineData("xamarintvos10", "portable-net451+netcore451")]
         [InlineData("xamarinwatchos10", "portable-net45+win8")]
+        [InlineData("xamarinwatchos10", "portable-net451+netcore451")]
         [InlineData("xamarinmac20", "portable-net45+win8")]
+        [InlineData("xamarinmac20", "portable-net451+netcore451")]
```

Comment:
Would you add tests for net451+win81  also? It should be the same as net451+netcore451.